sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_chat_completion.ipynb

```diff
@@ -486,6 +486,7 @@
     "        learning_rate=0.00002,\n",
     "        per_device_train_batch_size=1,\n",
     "        num_train_epochs=3,\n",
+    "        data_generation_task_type=\"NLI\",\n",
```

Comment:
Could we please make this parameter optional, if not already?

Reply:
@skasturi - If the parameter is optional, then the default data generation task is ambiguous. We have had a discussion about this and then it's marked as a mandatory parameter.